### PR TITLE
Save function (v03)

### PR DESCRIPTION
**Pull request by alexmaragko**
Save function (v03), and potential bug fix for duplicate model components.

### CODE REVIEW

File: pahfit/PAHFITBase.py [hunk 1]
```diff
@@ -225,12 +229,180 @@ def plot(self, ax, x, y, model):
         # print(obs_fit.param_names)
         # print(obs_fit.parameters)
 
-    def save(self, filename):
+    def save(self, obs_fit, filename, outform):
```
Comment (karllark) [inline pahfit/PAHFITBase.py]: Needs an updated docstring (info between """ and """).

**Comment by karllark** [comment]
Looks great!

**Comment by karllark** [comment]
Will (partially?) address issue #13.  And fully address #42.

File: pahfit/PAHFITBase.py (hunk 1 above)
Comment (alexmaragko) [reply pahfit/PAHFITBase.py]: Yes. How do we proceed with this? Update the docstring and then make a new pull request?

Comment (karllark) [reply pahfit/PAHFITBase.py]: Just update the docstring and push to the same branch.  The pull request will automatically update.  :-)

Comment (alexmaragko) [reply pahfit/PAHFITBase.py]: Cool! I'll do that.

**Comment by karllark** [comment]
Great.  Merging now.

**Pull request merged**